src/.github/workflows/ci.yml.jinja

```diff
@@ -116,7 +118,7 @@ jobs:
             context: .
             file: ./Dockerfile
             platforms: |
-            {%- for platform in platforms or ['linux/386','linux/amd64','linux/arm/v6','linux/arm/v7','linux/arm/v8','linux/arm64','linux/ppc64le','linux/s390x'] %}
+            {%- for platform in docker_platforms or ['linux/386','linux/amd64','linux/arm/v6','linux/arm/v7','linux/arm/v8','linux/arm64','linux/ppc64le','linux/s390x'] %}
```

Comment:
Just `platforms` or `image_platforms` is better.

Reply:
Thanks! Done